src/utils/Utils.ts

```diff
@@ -0,0 +1,5 @@
+import {sortMapByKey} from './dataStructureUtils';
+
+export {
```

Comment:
Do we really need the extra hop? 
Let's just import directly from the file we want 

(`import sortMapByKey from utils/dataStructureUtils)`
Typescript makes these imports really simple.